### ISSUE TITLE
enable 3D volume error

### ISSUE THREAD

**Issue by wilsao-cyber** [VirtualFlyBrain/VFB2#336]
Steps to reproduce the problem: 

Please fill the below with the necessary steps to reproduce the problem

Search for MBON 14 and click on the enabled 3D volume

Error Information:

e[n].isVisible is not a function

TypeError: e[n].isVisible is not a function
    at e.isVisible (https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:282:27258)
    at Object.condition (https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:298:276698)
    at n.value (https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:298:702057)
    at n.value (https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:298:702428)
    at https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:298:702399
    at Array.map (<anonymous>)
    at n.value (https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:298:702380)
    at https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:298:702693
    at Array.map (<anonymous>)
    at https://v2.virtualflybrain.org/org.geppetto.frontend/geppetto/build/main.bundle.js:298:702674

```diff
%2B Host for MessageSocket to connect: wss://v2.virtualflybrain.org/org.geppetto.frontend/GeppettoServlet%0A%2B %c WebSocket Status %2D Opened 
background: %23444; color: %23bada55%0A%2B 
 %c %c %c PixiJS 4.8.9 %2D &#10032; WebGL &#10032;  %c  %c  http://www.pixijs.com/  %c %c &#9829;%c&#9829;%c&#9829; 


background: %23ff66a5; padding:5px 0;
background: %23ff66a5; padding:5px 0;
color: %23ff66a5; background: %23030307; padding:5px 0;
background: %23ff66a5; padding:5px 0;
background: %23ffc3dc; padding:5px 0;
background: %23ff66a5; padding:5px 0;
color: %23ff2424; background: %23fff; padding:5px 0;
color: %23ff2424; background: %23fff; padding:5px 0;
color: %23ff2424; background: %23fff; padding:5px 0;%0A%2B Loading 171 slices/tiles...%0A%2D TypeError: e[n].isVisible is not a function
```

**Comment by Robbie1977** [comment]
Hi @wilsao-cyber we've been unable to replicate the error does it reoccur for you or was it a 1 off error?

**Comment by wilsao-cyber** [comment]
it's an error that happens whenever searching for "MBON14(a3)_R (FlyEM-HB:300972942)" and clicking the enabled 3D volume on the top of the term info.

**Comment by admclachlan** [comment]
We've found the issue and will have a fix out ASAP. Thank you for reporting!

**Comment by Robbie1977** [comment]
fix relased as part of v2.2.2.4; should be live now
